lib/events.rb

```diff
@@ -2,4 +2,11 @@
   event :UserChanged do |data|
     Bibliotheca::Collection::Users.import_from_json! data[:user]
   end
+
+  [:OrganizationCreated, :OrganizationChanged].each do |it|
+    event it do |payload|
+      organization = payload[:organization]
+      Bibliotheca::Collection::Organizations.upsert! organization if organization[:name] == 'base'
```

Comment:
`upsert!` method does not exist. It is called `upsert_by!` and it takes two arguments. 

I am fixing this right now and adding tests.